recipes/pytensor/meta.yaml

```
@@ -0,0 +1,109 @@
+{% set version = "2.8.10" %}
+
+package:
+  name: pytensor-suite  # https://github.com/conda/conda-build/issues/3933
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pytensor/pytensor-{{ version }}.tar.gz
+  sha256: 1cc26ebf6a5a8678b37bcb6ae497334bca3e3f7e27cb7815b0c9af0c0deb83f3
+  patches:
+
+build:
+  number: 0
+
+outputs:
+  - name: pytensor-base
+    build:
+      script:
+        - python -m pip install . --no-deps -vv
+      entry_points:
+        - pytensor-cache = bin.pytensor_cache:main
+    requirements:
+      build:
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+      host:
+        - pip
+        - python
+      run:
+        - python
+
+        # Ref: <https://github.com/pymc-devs/pytensor/blob/rel-{{ version }}/setup.py>
+        - numpy>=1.17.0
+        - scipy>=0.14
```

Sometimes this fails, not sure why it didn't here but better safe than sorry.

```suggestion
        - numpy >=1.17.0
        - scipy >=0.14
```